tests/lib/rules/jsx-indent.js

```diff
@@ -1883,5 +1926,46 @@ const Component = () => (
     errors: [
       {message: 'Expected indentation of 8 space characters but found 4.'}
     ]
+  }, {
+    code: [
+      '<div>',
+      'text',
+      '</div>'
+    ].join('\n'),
+    errors: [
+      {message: 'Expected indentation of 4 space characters but found 0.'}
+    ]
+  }, {
+    code: [
+      '<div>',
+      '  text',
+      'text',
+      '</div>'
+    ].join('\n'),
+    errors: [
+      {message: 'Expected indentation of 4 space characters but found 2.'},
+      {message: 'Expected indentation of 4 space characters but found 0.'}
+    ]
+  }, {
+    code: [
+      '<div>',
+      '\t  text',
+      '  \t  text',
+      '</div>'
+    ].join('\n'),
+    errors: [
+      {message: 'Expected indentation of 4 space characters but found 0.'},
+      {message: 'Expected indentation of 4 space characters but found 2.'}
+    ]
+  }, {
+    code: [
+      '<div>',
+      '\t\ttext',
+      '</div>'
+    ].join('\n'),
+    options: ['tab'],
+    errors: [
+      {message: 'Expected indentation of 1 tab character but found 2.'}
+    ]
```

Comment:
A test with `babel-eslint` is needed here.

Reply:
Thanks!! I agree. added a test with `babel-eslint`.